src/docker.test.ts

```diff
@@ -48,34 +55,69 @@ describe("Docker images", (): void => {
   const assertLoadDockerImages = (cacheHit: boolean): void => {
     expect(core.getInput).lastCalledWith("key", { required: true });
     expect(cache.restoreCache).lastCalledWith([docker.DOCKER_IMAGES_PATH], KEY);
-    expect(core.saveState).lastCalledWith(docker.CACHE_HIT, cacheHit);
+    expect(core.saveState).nthCalledWith<[string, boolean]>(
+      1,
+      docker.CACHE_HIT,
+      cacheHit
+    );
     expect(core.setOutput).lastCalledWith(docker.CACHE_HIT, cacheHit);
+    expect(util.execBashCommand).toHaveBeenCalledTimes(1);
```

Comment:
Let's assert the call count after the corresponding called withs for consistency with the rest of the tests and because the resulting error message is less specific when the call count is off than when the arguments don't match.